### PR TITLE
ContextToYML script

### PR DESCRIPTION
**Pull request by orenzohar**
## Status
Ready

## Description
This script parses a Context output into yml formatted file with the context path of each field. 

### run a command
image
### gets its context output
image
### hit 
```!ContextToYML command_name=intsights-get-alert-by-id json_file_as_text=`THE OUTPUT` ```
#### get a formated yml file
image

### CODE REVIEW

File: Scripts/ContextToYML/ContextToYML.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
+from CommonServerPython import *
```
[the remark this answers is not in the document]
Comment (orenzohar) [reply Scripts/ContextToYML/ContextToYML.py]: works fine without it but ok

File: Releases/LatestRelease/ContextToTML.md [hunk 2]
```diff
@@ -0,0 +1 @@
+script that turns context output into yml formatted file with the context path of each field.
```
[the remark this answers is not in the document]
Comment (orenzohar) [reply Releases/LatestRelease/ContextToTML.md]: thanks

**Comment by orenzohar** [comment]
> how do you handle fields that are not always there? (`createContext(output, removeNulls=True)`)
> you can't cover them so this should be a "known limitation in the docs"

I agree

**Comment by Itay4** [comment]
This shouldn’t be in the scripts directory, as there is no reason to release this

**Comment by yaakovi** [comment]
@avidan-H FYI

**Pull request closed without merging**